primitives/src/parachain.rs

```diff
@@ -580,12 +460,6 @@ sp_api::decl_runtime_apis! {
 		fn parachain_status(id: Id) -> Option<Status>;
 		/// Get the given parachain's head code blob.
 		fn parachain_code(id: Id) -> Option<Vec<u8>>;
-		/// Get all the unrouted ingress roots at the given block that
-		/// are targeting the given parachain.
-		///
-		/// If `since` is provided, only messages since (including those in) that block
-		/// will be included.
-		fn ingress(to: Id, since: Option<BlockNumber>) -> Option<StructuredUnroutedIngress>;
```

Comment:
cc @bkchr will this brick existing chains without a runtime API version upgrade?

Reply:
Looks like it was only called in conjunction with Parachain validation? So, as we don't have any Parachain, there should be no problem.